test-jest/cartesian/Bar.spec.tsx

```diff
@@ -65,29 +66,29 @@ describe('<Bar />', () => {
     ];
 
     it('Will create a background Rectangle with the passed in props', () => {
-      const wrapper = render(
+      const { container } = render(
         <Surface width={500} height={500}>
-          <Bar data={composedDataWithBackground} background={{ fill: '#000' }} />
+          <Bar data={composedDataWithBackground} background={{ fill: '#000' }} dataKey="value" />
         </Surface>,
       );
 
-      expect(wrapper.find('.recharts-bar-background-rectangle').length).to.equal(2);
+      expect(container.querySelectorAll('.recharts-bar-background-rectangle')).toHaveLength(
+        composedDataWithBackground.length,
+      );
     });
 
     it('Will accept a function for the background prop', () => {
-      const wrapper = mount(
+      const className = 'test-custom-background';
+      const backgroundComponent = () => {
+        return <div key={_.uniqueId()} className={className} />;
+      };
+      const { container } = render(
```

Comment:
Nit: It could be helpful, and improve readability, if the arrange/act/assert sections of each test were separated by an empty line. 
```suggestion

      const { container } = render(
```